src/patterns/ec2-app/base.ts

```diff
@@ -290,7 +291,8 @@ function restrictedCidrRanges(ranges: IPeer[]) {
  * ```
  *
  * If you _must_ support a non-standard VPC setup, it's possible to override the VPC and subnet selection into which
- * instances will be started. This is done by setting the `vpc` and `privateSubnets` props.
+ * instances will be started. This is done by setting the `vpc` and `privateSubnets` props.  You can also override
```

Comment:
I wonder if we could be a bit clearer here that if you set one of these you need to set all of them?

Reply:
👍 Updated